packaging/installer/kickstart.sh

```diff
@@ -772,15 +772,64 @@ claim() {
+      return 0
+      ;;
+    8)
+      warning "Failed to claim node due to an invalid agent ID. You can usually resolve this by removing ${INSTALL_PREFIX}/var/lib/netdata/registry/netdata.public.unique.id and restarting the agent, then trying to claim it again."
+      ;;
+    9)
+      warning "Failed to claim node due to an invalid node name. This probably means you tried to specify a custom name for this node (for example, using the --claim-hostname option), but the hostname itself was either empty or consisted solely of whitespace. You can resolve this by specifying a valid host name and trying again."
+      ;;
+    10)
+      warning "Failed to claim node due to an invalid room ID. Please check that the room or rooms you are trying to add the node to exists and that the list of rooms provided to the --claim-rooms option ('${NETDATA_CLAIM_ROOMS}') matches what you saw in the Cloud and try again."
+      ;;
+    11)
+      warning "Failed to claim node due to an issue with the generated RSA key pair. You can usually resolve this by removing all files in ${INSTALL_PREFIX}/var/lib/netdata/cloud.d and then trying again."
+      ;;
+    12)
+      warning "Failed to claim node due to an invalid or expired claiming token. Please check that the toekn specified with the --claim-token option ('${NETDATA_CLAIM_TOKEN}') matches what you see in the Cloud and try again."
+      ;;
+    13)
+      warning "Failed to claim node because the Cloud thinks it is already claimed. If this node was created by cloning a VM or as a container from a template, please remove the file ${INSTALL_PREFIX}/var/lib/netdata/registry/netdata.public.unique.id and restart the agent, then try to claim it again. Otherwise, if you are certain this node has never been claimed before, you can use the --claim-id option to specify a new node ID to use for claiming, for example by using the uuidgen command like so: --claim-id \"\$(uuidgen)\""
+      ;;
+    14)
+      warning "Failed to claim node because the node is already in the process of being claimed. You should not need to do anything to resolve this, the node should show up properly in the Cloud soon. If it does not, please report a bug."
+      ;;
+    15|16|17)
+      warning "Failed to claim node due to an internal server error in the Cloud. Please retry claiming this node later, and if you still see this message file a bug report."
+      ;;
+    18)
+      warning "Unable to claim node because this Netdata installation does not have a unique ID yet. Make sure the agent is running and started up correctly, and then try again."
+      ;;
+    *)
+      warning "Failed to claim node for an unknown reason. This usually means either networking problems or a bug. Please retry claiming later, and if you still see this message file a bug report."
```

Comment:
Can we add a link to submit a bug as well? https://github.com/netdata/netdata/issues/new?assignees=&labels=bug%2Cneeds+triage&template=BUG_REPORT.yml&title=%5BBug%5D%3A+